packages/components/Tabs/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@fluentui-react-native/tabs",
+  "version": "0.1.0",
+  "description": "A cross-platform Tabs component using the Fluent Design System",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/button": "^0.17.9",
```

Comment:
super nit: we tend to try and keep these in alphabetical order

Reply:
Running `yarn add <package>` at the `packages/components/Tab` directory will also auto sort it for you